Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -282,6 +282,22 @@ def run_checks(
           version_info[:latest] if version_info.present?
         end
 
+        # Check current and latest resources (if "--resources" flag is given)
+        # Only check current and latest versions if we have resources to check against
+        check_for_resources = check_resources && formula_or_cask.is_a?(Formula) && formula_or_cask.resources.present?
+        if check_for_resources
+          resource_version_info = formula_or_cask.resources.map do |resource|
+            resource_info = resource_version(
+              resource,
+              json:      json,
+              full_name: use_full_name,
+              verbose:   verbose,
+              debug:     debug,
+            )
+            resource_info
```

Comment:
```suggestion
            resource_version(
              resource,
              json:      json,
              full_name: use_full_name,
              verbose:   verbose,
              debug:     debug,
            )
```
It might be sufficient to do just this since the `resource_info` variable is practically useless – it's just used to return the value that was assigned to it in the previous statement.